internal/provider/resource_aws_network_peering.go

```diff
@@ -66,14 +66,14 @@ func resourceAwsNetworkPeering() *schema.Resource {
 					return strings.ToLower(old) == strings.ToLower(new)
 				},
 			},
+			// Optional inputs
```

Comment:
Formerly we agreed to keep that change out of this PR but happy to comply whatever makes sense to you. I am going on call tomorrow so will defer to @antonakv for handling the complete removal of these fields and adding the ID 

I think making another feature branch and pointing this branch to the feature branch makes sense, but will see how Anton would like to handle it.

Reply:
let's probably keep this PR focused on HVN routes only and do the rest in a separate PR because it would require updating docs, examples, tests, etc

Reply:
Agreed, separate PRs for each change would be best. So you'll move these changes to peering and TGW into a new PR?

Reply:
For the purpose of this PR the changes to peering and TGW is necessary, otherwise submitting a value during creation of peering/tgw attachment will result in a route having been created for these resources, causing an error during later HVN route creation

Reply:
removed all not HVN route related changed from this PR